README.md

```diff
@@ -29,6 +29,8 @@ Supported metrics right now as follows.
 
 Global:
 
+ * _System/Fortimanager/Status_
```

Comment:
Isn't this per-VDOM?

Reply:
I see that it was global first, so this is probably a left-over.

Reply:
Yes I made a mistake querying a non multiVDOM firewall. So I though this was a global metric, but this actually a per-VDOM.
Will fix that with next correction